src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,24 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => <>Put the comment here</>;
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
```

Comment:
Read this article about React.FC https://dev.to/elhamnajeebullah/react-typescript-what-is-reactfc-and-why-should-i-use-it-4029

Reply:
All I concluded from the article was that React FC is no recommended idk what i should fix